cmd/limactl/list.go

```diff
@@ -22,9 +35,18 @@ func newListCommand() *cobra.Command {
 		ValidArgsFunction: cobra.NoFileCompletions,
 	}
 
+	listCommand.Flags().StringP("format", "f", "", "Format the output using the given Go template")
 	listCommand.Flags().Bool("json", false, "JSONify output")
 	listCommand.Flags().BoolP("quiet", "q", false, "Only show names")
 
+	fieldsCommand := &cobra.Command{
+		Use:   "fields",
+		Short: "Fields available for format",
+		Long:  strings.Join(instanceFields(), "\n"),
+	}
+
+	listCommand.AddCommand(fieldsCommand)
```

Comment:
I don't think this is correct.
We can just have a documentation to explain the fields.

Reply:
It uses https://pkg.go.dev/github.com/spf13/cobra#Command.IsAdditionalHelpTopicCommand

Reply:
```
List instances of Lima.

Usage:
  limactl list [flags]

Aliases:
  list, ls

Flags:
  -f, --format string   Format the output using the given Go template
  -h, --help            help for list
      --json            JSONify output
  -q, --quiet           Only show names

Global Flags:
      --debug   debug mode

Additional help topics:
  limactl list fields Fields available for format

```

Reply:
The problem is that this makes "fields" to be a reserved instance name (in conjunction with #413).

Reply:
There is probably some other way to do it in cobra

Reply:
> The problem is that this makes "fields" to be a reserved instance name (in conjunction with #413).

I would expect it to work if you use `limactl ls -- fields`, but it is an ugly exception.

I think I would prefer an additional option: `limactl ls --list-fields` to show the fields (that is mutually exclusive with `--format` and `--json`).